reference/extensions/package_signing.rst

```diff
@@ -0,0 +1,45 @@
+.. _reference_extensions_package_signing:
+
+Package signing
+---------------
+
+This plugin, which must be located in the cache ``extensions/plugins/sign/sign.py`` file contains 2 methods:
+
+- The ``sign(ref, artifacts_folder, signature_folder)`` executes for every recipe and package that is to
+  be uploaded to a server. The ``ref`` is the full reference to the artifact, it can be either a recipe 
+  reference or a package reference. The ``artifacts_folder`` is the folder containing the files to be 
+  uploaded, typically the ``conanfile.py``, ``conan_package.tgz``, ``conanmanifest.txt``, etc. The
+  ``signature_folder`` contains the folder in which the generated files should be written.
```

Comment:
is `signature_folder` uploaded to the server as well?
asking, cause in the opposite case, I want be able to upload and sign from one machine, and then download and verify signature on another machine

Reply:
Yes, it is uploaded to server.